assets/queries/cloudFormation/filter_and_alarm_exist_for_cloudtrail_config_changes/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_CloudTrail_Configuration_Changes",
+  "queryName": "Filter And Alarm Not Exists For CloudTrail Configuration Changes",
```

Comment:
Change **directory name** and **id** to: `cloudtrail_configuration_changes_without_alarms`
Change **query name** to: `Cloudtrail Configuration Changes Without Alarms`